canarieapi-cron

```diff
@@ -1,2 +1,2 @@
-* * * * * root python -c 'from canarieapi import logparser; logparser.cron_job()' >> /var/log/logparser_cron.log 2>&1
-* * * * * root python -c 'from canarieapi import monitoring; monitoring.cron_job()' >> /var/log/monitoring_cron.log 2>&1
+* * * * * root /usr/local/bin/python3 -c 'from canarieapi import logparser; logparser.cron_job()' >/proc/1/fd/1 2>/proc/1/fd/2
+* * * * * root /usr/local/bin/python3 -c 'from canarieapi import monitoring; monitoring.cron_job()' >/proc/1/fd/1 2>/proc/1/fd/2
```

Comment:
This is documented in docker-compose.yml (which clarifies the usage of nginx-log.conf):

```
      # This nginx config file ensure that the nginx container will write the access logs to stdout as well as a file
      # on disk. This is required because canarie-api-cron expects a regular file to parse logs from.
```

and in default-configuration.py

```
# If this is True, canarie-api will parse the nginx logs in DATABASE["access_log"] and report statistics
```

I think it will be confusing to reference that file here in the cron file since nothing here references nginx. I can add extra comments to the monitoring.py script if you think we need additional clarification.